.github/workflows/smoke-tests.yml

```diff
@@ -168,20 +158,20 @@ jobs:
         run: |
           CLIENT_ID=$(echo $CLIENT_CREDENTIALS_PARAMS | jq -r .client_id)
           CLIENT_SECRET=$(echo $CLIENT_CREDENTIALS_PARAMS | jq -r .client_secret)
-          docker compose -f docker-compose.test.yml run --rm postman_test test/postman_test/BCDA_Postman_Smoke_Tests.postman_collection.json \
```

Comment:
I think it's generally a good idea to use `docker compose run` since it includes configuration from the compose file (which is empty now, but might not be in the future). I'd also keep the `--rm` so that the containers are removed after they run and don't stack up on our local machines.